vms/avm/vm.go

```diff
@@ -579,7 +557,7 @@ func (vm *VM) initState(tx *txs.Tx) {
 		zap.Stringer("txID", txID),
 	)
 	vm.state.AddTx(tx)
-	vm.state.AddStatus(txID, choices.Accepted)
+	vm.state.DeleteStatus(txID)
```

Comment:
We assume that transactions on disk with no status are accepted, so we just need to make sure to delete any previously existing statuses on disk.